src/openvslam/tracking_module.cc

```diff
@@ -135,9 +143,28 @@ Mat44_t tracking_module::track_RGBD_image(const cv::Mat& img, const cv::Mat& dep
     const auto end = std::chrono::system_clock::now();
     elapsed_ms_ = std::chrono::duration_cast<std::chrono::milliseconds>(end - start).count();
 
+    if (!tracking_started_) {
+        tracking_started_ = true;
+    }
+
     return curr_frm_.cam_pose_cw_;
 }
 
+bool tracking_module::track_pose(const Mat44_t& pose) {
+    curr_frm_.set_cam_pose(pose);
+
+    const auto candidates = map_db_->get_close_keyframes(pose, cfg_->update_pose_keyframes_);
```

Comment:
Error checking that this is non-zero would be good

Reply:
Done in https://github.com/OpenVSLAM-Community/openvslam/pull/79/files#diff-9d7c6e92591d28cb8c87b3973729fddb3abd564e9feae8861851fd48dd31b2caR369